subprojects/logging/src/test/groovy/org/gradle/internal/logging/sink/GroupingProgressLogEventGeneratorTest.groovy

```diff
@@ -20,17 +20,31 @@ import org.gradle.api.logging.LogLevel
 import org.gradle.internal.logging.OutputSpecification
```

Comment:
We'll also need at least one functional test that actually renders the output that we can verify.